### PR TITLE
[frontend] Fix malware analyses unknown display

### LINKED ISSUE #6010: Malware analyses displayed as Unknown in filters list
## Description
Malware analyses are displayed as unknown in some locations: 
- filters in triggers

## Reproducible Steps

Create an instance trigger. Open the 'related entity' filter. Malware analyses are displayed as Unknown:

Create a trigger. Open the 'contains' filter. Malware analyses are displayed as Unknown:
image

### PR DESCRIPTION
**Pull request by Kedae**
Closes #6010

### CODE REVIEW

File: opencti-platform/opencti-front/src/utils/Graph.js [hunk 1]
```diff
@@ -394,6 +394,7 @@ export const defaultValue = (n, fallback = 'Unknown') => {
       || n.attribute_abstract
       || n.opinion
       || n.value
+      || n.product
```
Comment (Archidoit) [inline opencti-platform/opencti-front/src/utils/Graph.js]: it's result_name that we want to display for malware analyses

**Comment by Archidoit** [comment]
we should rather add ...on MalwareAnalyses {result_name} in the stix core object query of useSearchEntities.tsx

**Review by Archidoit — CHANGES_REQUESTED** [review]
display the malware analyses result_name, not its product

File: opencti-platform/opencti-front/src/utils/Graph.js (hunk 1 above)
Comment (Kedae) [reply opencti-platform/opencti-front/src/utils/Graph.js]: My bad, missed that. I assumed that the spread on the query was correct 😅
I'll fix that

**Comment by Archidoit** [comment]
malware analyses backend representative is not correct (display the id instead of the result_name) : 
image

--> should add the malware analyses case in extractEntityRepresentativeName() of entity-representative.js

**Review by Archidoit — APPROVED** [review]

**Pull request merged**